### PR TITLE
add back goveralls

### LINKED ISSUE #954: Use Coverage to stat test coverage
In https://github.com/kubeflow/katib, they have a badge of coverage. How about we add one as well?

overage

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/sql-machine-learning/sqlflow/issues/954

the environment variable is already in `.travis.yml`:
```
env:
   global:
     secure:
```

### CODE REVIEW

File: scripts/test_units.sh [hunk 1]
```diff
@@ -40,5 +40,13 @@ go install ./...
 # -p 1 is necessary since tests in different packages are sharing the same database
 # ref: https://stackoverflow.com/a/23840896
 SQLFLOW_log_level=debug go test -v -p 1 ./...  -covermode=count -coverprofile=coverage.out
+if [[ "$TRAVIS_BRANCH" == "develop" ]]; then
+    if [[ "$TRAVIS_EVENT_TYPE" != "cron" ]]; then
+        # upload to coveralls.io only when commit to develop branch
+        goveralls -coverprofile=coverage.out -service=travis-ci -repotoken $COVERALLS_TOKEN
```
Comment (Yancey0623) [inline scripts/test_units.sh]: How about moving these to `deploy.sh`?

Comment (typhoonzero) [reply scripts/test_units.sh]: Done

File: scripts/test_units.sh [hunk 2]
```diff
@@ -41,4 +41,6 @@ go install ./...
 # ref: https://stackoverflow.com/a/23840896
 SQLFLOW_log_level=debug go test -v -p 1 ./...  -covermode=count -coverprofile=coverage.out
 
+
```
Comment (Yancey0623) [inline scripts/test_units.sh]: please remove the unnecessary blank line.

**Review by Yancey0623 — DISMISSED** [review]
LGTM with a tiny comment.

**Comment by weiguoz** [comment]
The badge: https://coveralls.io/repos/github/sql-machine-learning/sqlflow/badge.svg is grey ( "coverage unknown").
Is it will be fixed once the cron-job pushes coverage status to coveralls.io?

File: scripts/build_docker_image.sh [hunk 3]
```diff
@@ -57,6 +57,10 @@ go get golang.org/x/lint/golint
 mv $GOPATH/bin/golint /usr/local/bin
 go get golang.org/x/tools/cmd/goyacc
 mv $GOPATH/bin/goyacc /usr/local/bin/
+go get golang.org/x/tools/cmd/cover
```
Comment (tonyyang-svail) [inline scripts/build_docker_image.sh:60]: Do we need to install this? I think `cover` is part of the gotools.

**Comment by typhoonzero** [comment]
@weiguoz I've tested to push the coverage information to coveralls.io after this PR is merged, the badge should be updated.

File: scripts/build_docker_image.sh (hunk 3 above)
Comment (typhoonzero) [reply scripts/build_docker_image.sh:60]: This is following https://docs.coveralls.io/go, I think it's better to keep it here.

**Review by weiguoz — APPROVED** [review]
LGTM

**Pull request merged**